client/src/Components/Pages/Dashboard/MyActivety/style.css

```diff
@@ -0,0 +1,28 @@
+.done{
+  color:green;
+  font-weight: bold;
+  display: flex;
+  margin-top: 5px;
+}
+.not-done{
+color: red;
+font-weight: bold;
+display: flex;
+margin-top: 5px;
```

Comment:
.done and .not-done have many attributes in common (font-weight, display: flex, etc.), so it might be helpful to combine those shared styles in one place instead of having them duplicated.  For example, you could have them in `.my-activity > div`, or you could add another class to all of those activity divs (i.e. `<div className='activity done'>` and `<div className='activity not-done'>`).  Then .done and .not-done would just contain the alternate red/green color.

Removing the duplication helps avoid bugs - when the styles are duplicated, there's always the danger of changing one but forgetting to change the other.

Reply:
That's is very useful.